app/components/main-content/ThankYou.tsx

```diff
@@ -1,76 +1,38 @@
-        <span className="align-text-bottom">Thank you!</span>
-        <Player
-          autoplay
-          loop={false}
-          src={heartAnimationData}
-          style={{
-            width: '50px',
-            height: '50px',
-            background: 'transparent',
-            fillOpacity: '0',
-          }}
-        >
-          <Controls
-            visible={false}
-            buttons={['play', 'repeat', 'frame', 'debug']}
-          />
-        </Player>
-      </h1>
+    <article aria-live="polite" className="py-12 ml-4 mb-2 px-2">
+      <h2 className="flex pt-4 h-[60px] align-bottom">
+        <span className="font-serif text-3xl font-bold align-text-bottom">
+          Thank you!
+        </span>
+      </h2>
       <section>
-        <p className="pt-4 text-lg">
-          We&apos;re thrilled to have you join us on this exciting journey!
-        </p>
-        <p className="pt-2 text-xs">
-          (Our demo is almost ready, and we can&apos;t wait for you to
-          experience it firsthand.)
-        </p>
+      <p className="pt-4 text-lg">We&apos;re thrilled to have you join us on this exciting journey!</p>
+      <p className="text-lg">Keep an eye on your inbox for an <strong>exclusive invitation</strong> coming your way soon!</p>
+      <p className="hidden text-xs pt-2">(Our demo is almost ready, and we can&apos;t wait for you to experience it firsthand.)</p>
       </section>
       <div>
         <Player
           autoplay
           loop={true}
           src={mailAnimationData}
-          style={{ width: '300px', height: '300px' }}
+          style={{ width: "300px", height: "200px" }}
         >
-          <Controls
-            visible={false}
-            buttons={['play', 'repeat', 'frame', 'debug']}
-          />
-        </Player>
+          <Controls visible={false} buttons={['play', 'repeat', 'frame', 'debug']} />
+        </Player>        
       </div>
       <section>
-        <p>
-          If you&apos;d like to keep up with the project,{' '}
-          <a
-            href="https://www.linkedin.com/company/100947448/"
-            target="_blank"
-          >
-            follow us on LinkedIn!
-          </a>
-        </p>
-      </section>
-      <section>
-        <p className="text-lg">
-          Keep an eye on your inbox for an <strong>exclusive invitation</strong>{' '}
-          coming your way soon!
-        </p>
-        <p className="absolute bottom-10 text-xs">
-          Got questions? We&apos;re here to help! Shoot us an email at{' '}
-          <a href={`mailto:${linktaEmail}`}>{linktaEmail}</a>
-        </p>
+        <p>If you&apos;d like to keep updated with the project, <a href='https://www.linkedin.com/company/100947448/' className="underline decoration-light-link font-semibold">follow us on LinkedIn!</a> or Email us at <a href={`mailto:${linktaEmail}`} className="underline decoration-light-link font-semibold">{linktaEmail}</a> </p>
```

Comment:
It has quite some styling involved so I'll leave the update to you.

Reply:
Just look through the file, the changes were intentional as thank you page was built based off an older version of the submission status page. Since this PR was intended to merge last, we decided to overwrite this page with the last PR.

Reply:
Please double check the content. This current version contains typos which was corrected in the content being overwritten.

Reply:
Email -> email

Reply:
Oh my eyes... I kept checking the email address and couldn't find the typo. Thank you @kinskiwu